### DISCUSSION TITLE
Exclude Views (AspNetCoreGeneratedDocument) folder from coverage coverlet

### DISCUSSION THREAD

**Discussion by yacine-karim** [coverlet-coverage/coverlet#1374] [Q&A] [answered]
Hello everyone

I have this class **AspNetCoreGeneratedDocument.Views_Users__AddOrUpdateUser** appearring in the code coverage result 

_Assembly_: **Solution.Web**
_Class_: **AspNetCoreGeneratedDocument.Views_Users__AddOrUpdateUse**

When i click on the class to check it, it takes me to the **_AddOrUpdateUser** View in the view folder

Path: **D:\a\1\s\Solution.Web\Views\Users\_AddOrUpdateUser.cshtml**

I want to exclude this folder using the **.runsettings** file:

```
<?xml version="1.0" encoding="utf-8" ?>
<RunSettings>
	<DataCollectionRunSettings>
		<DataCollectors>
			<DataCollector friendlyName="XPlat code coverage">
				<Configuration>
					<!-- Globbing filter -->
					<ExcludeByFile>**/Solution.Web/Views/*.cshtml</ExcludeByFile>
					<IncludeTestAssembly>false</IncludeTestAssembly>
					<DeterministicReport>false</DeterministicReport>
				</Configuration>
			</DataCollector>
		</DataCollectors>
	</DataCollectionRunSettings>
</RunSettings>
```

**Comment by MarcoRossignoli** [comment, marked as answer]
I see the correct issue here https://github.com/coverlet-coverage/coverlet/issues/1375

**Comment by Bertk** [comment]
Closing this discussion since it’s no longer relevant to the current state of the project.
